tests/storage/hotplug.go

```diff
@@ -514,6 +515,19 @@ var _ = SIGDescribe("Hotplug", func() {
 		verifyVolumeNolongerAccessible(vmi, targets[0])
 	}
 
+	verifyNoAttachVolume := func(vm *v1.VirtualMachine, addVolumeFunc addVolumeFunction, sc string, volumeMode k8sv1.PersistentVolumeMode) {
+		dv := createDataVolumeAndWaitForImport(sc, volumeMode)
+
+		vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		libwait.WaitForSuccessfulVMIStart(vmi,
+			libwait.WithTimeout(240),
+		)
+
+		addVolumeFunc(vm.Name, vm.Namespace, "testvolume", dv.Name, v1.DiskBusSCSI, true, "")
+		verifyNoVolumeAttached(vmi, "testvolume")
+	}
+
```

Comment:
Do we really need this function? Can't we just do it inside the DescribeTable body function?
Also, the function does more than the name expose.
I'd suggest you move the content of this to the DescribeTable. WDYT?

Reply:
This night I woke up from a nightmare and then I knew I had to do this. +1